spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/HyperConformance.scala

```diff
@@ -157,5 +161,21 @@ object HyperConformance extends StreamTransformerFactory {
     }
   }
 
+  private def getReportDate(conf: Configuration): String = {
+    if (conf.containsKey(reportDateKey)) {
+      conf.getString(reportDateKey)
+    } else {
+      DateTime.now().toLocalDate.toString(Constants.ReportDateFormat)
```

Comment:
If you use just `ReportDateFormat` you don't have to import constants twice. Or switch line 53-56 to this way of using constants.